wdl/BAFTestChromosome.wdl

```diff
@@ -114,28 +117,22 @@ task BAFTest {
     chrom=$(cut -f1 ~{bed} | head -n1)
     set -o pipefail
 
-    # Temporary workaround for corrupted tabix downloads
-    x=0
-    while [ $x -lt ~{tabix_retries} ]
-    do
-      # Download twice
-      GCS_OAUTH_TOKEN=`gcloud auth application-default print-access-token` tabix ~{baf_metrics} "$chrom":"$start"-"$end" | bgzip -c > local_baf_1.bed.gz
-      GCS_OAUTH_TOKEN=`gcloud auth application-default print-access-token` tabix ~{baf_metrics} "$chrom":"$start"-"$end" | bgzip -c > local_baf_2.bed.gz
-      # Done if the downloads were identical, otherwise retry
-      cmp --silent local_baf_1.bed.gz local_baf_2.bed.gz && break       
-      x=$(( $x + 1))
-    done
-    echo "BAF-tabix retry:" $x
-    cmp --silent local_baf_1.bed.gz local_baf_2.bed.gz || exit 1
-
-    mv local_baf_1.bed.gz local_baf.bed.gz
+    java -Xmx~{java_mem_mb}M -jar ${GATK_JAR} LocalizeSVEvidence \
+      --sequence-dictionary ~{ref_dict} \
+      --evidence-file ~{baf_metrics} \
+      -L "${chrom}:${start}-${end}" \
+      -O local_baf.bed
+
+    # GATK does not block compress
```

Comment:
Again, I think it should be able to -- you might want to let the engine team know if you're having trouble with that.